### PR TITLE
Patch 7

### PR DESCRIPTION
**Pull request by Lovina-Saldanha**
Added a new section "Use the new EAC to enable or disable Outlook on the web"

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Exchange/ExchangeOnline/recipients-in-exchange-online/manage-user-mailboxes/enable-or-disable-outlook-web-app.md [hunk 1]
```diff
@@ -42,7 +42,23 @@ For additional management tasks related to managing email client access to a mai
...
 
-## Use the EAC to enable or disable Outlook on the web
+## Use the new EAC to enable or disable Outlook on the web
+1. In the new EAC, navigate to **Recipients** \> **Mailboxes**.
+
+2. In the list of user mailboxes, click the mailbox that you want to enable or disable Outlook on the web. A flyout is displayed for the selected user mailbox.
+
+3. Under the **Mailbox** properties link, click **Manage email apps settings** under **Email apps**.
+
+4. A flyout **Manage settings for email apps** is displayed. Do one of the following.
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/recipients-in-exchange-online/manage-user-mailboxes/enable-or-disable-outlook-web-app.md:52]:
```suggestion
4. A flyout **Manage settings for email apps** is displayed. Do one of the following:
```

File: Exchange/ExchangeOnline/recipients-in-exchange-online/manage-user-mailboxes/enable-or-disable-outlook-web-app.md [hunk 2]
```diff
@@ -42,7 +42,23 @@ For additional management tasks related to managing email client access to a mai
 > [!TIP]
 > Having problems? Ask for help in the Exchange forums. Visit the forums at [Exchange Online](https://social.technet.microsoft.com/forums/msonline/home?forum=onlineservicesexchange) or [Exchange Online Protection](https://social.technet.microsoft.com/forums/forefront/home?forum=FOPE).
 
-## Use the EAC to enable or disable Outlook on the web
+## Use the new EAC to enable or disable Outlook on the web
+1. In the new EAC, navigate to **Recipients** \> **Mailboxes**.
+
+2. In the list of user mailboxes, click the mailbox that you want to enable or disable Outlook on the web. A flyout is displayed for the selected user mailbox.
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/recipients-in-exchange-online/manage-user-mailboxes/enable-or-disable-outlook-web-app.md:48]:
```suggestion
2. In the list of user mailboxes, click the mailbox for which you want to enable or disable Outlook on the web. A flyout is displayed for the selected user mailbox.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by scanum** [comment]
@Kateyanne PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**